model/client4.go

```diff
@@ -3247,6 +3247,19 @@ func (c *Client4) UpdateConfig(config *Config) (*Config, *Response) {
 	return ConfigFromJson(r.Body), BuildResponse(r)
 }
 
+// MigrateConfig will migrate existing config with the new one.
```

Comment:
Minor nit:  I suggest `MigrateConfig will migrate existing config to the new one.`